### PR TITLE
footer: social media icons, extra links

### LINKED ISSUE #903: footer
- [x] add standard footer to /statistics
- [ ] add footer to any other page?
- [ ] make it look good
- [x] add standard footer to /messages -- already there
- [x] add standard footer to /meet? (that, or remove existing footer) -- already there
- [x] postponed till [consensus](https://meta.trustroots.org/t/show-last-date-the-code-was-changed/87): something like GitHub deploy date, date of most recent commit is good enough?
- [x] link to version on github, 
- [x] social media icons
- [x] at least links to: /contact, /faq, /foundation, /volunteering, /rules, /donate, /privacy
- [x] language selector, for now just a [link to meta](https://meta.trustroots.org/t/lets-translate-trustroots/59/3)
- [ ] _anything else?_

Here's some good stuff:
https://www.orbitmedia.com/blog/website-footer-design-best-practices/

### PR DESCRIPTION
**Pull request by guaka**
#### Proposed Changes

* see #903 

#### Testing Instructions

* e.g. http://localhost:3000/donate

Fixes #903

### CODE REVIEW

File: modules/core/server/views/partials/footer.server.view.html [hunk 1]
```diff
@@ -2,31 +2,47 @@
...
+  <hr>
+  <div class="navbar">
+    <div class="col-xs-6">
+      <ul class="navbar">
+        <li itemscope itemtype="http://schema.org/Organization">
+          <a ui-sref="foundation" itemprop="name">
+            Trustroots Foundation
+          </a>
+          <link itemprop="url" href="https://www.trustroots.org/">
+          <link itemprop="logo" href="https://www.trustroots.org/img/logo/color.png">
```
Comment (simison) [inline modules/core/server/views/partials/footer.server.view.html]: We can render `https://www.trustroots.org` dynamically with `{{hostPort}}` 

See examples from `modules/core/server/views/layout.server.view.html`

File: modules/core/server/views/partials/footer.server.view.html [hunk 2]
```diff
@@ -2,31 +2,47 @@
...
+        </li>
+        <li><a ui-sref="privacy">Privacy</a></li>
+        <li><a ui-sref="contact">Contact</a></li>
+        <li><a ui-sref="faq">FAQ</a></li>
+        <li><a ui-sref="volunteering">Volunteering</a></li>
+        <li><a ui-sref="rules">Rules</a></li>
+        <li><a ui-sref="rules">Rules</a></li>
```
Comment (simison) [inline modules/core/server/views/partials/footer.server.view.html]: Rules is twice here.

File: modules/core/server/views/partials/footer.server.view.html [hunk 3]
```diff
@@ -2,31 +2,47 @@
...
-        Trustroots Foundation
-      </a>
-      <link itemprop="url" href="https://www.trustroots.org/">
-      <link itemprop="logo" href="https://www.trustroots.org/img/logo/color.png">
-
-      <ul class="list-inline text-center">
+  <hr>
+  <div class="navbar">
+    <div class="col-xs-6">
+      <ul class="navbar">
```
Comment (simison) [inline modules/core/server/views/partials/footer.server.view.html]: I don't think `navbar` has any effect here?

The list would work nicely without bulletins:

image

Using `list-unstyled` does the trick (https://getbootstrap.com/docs/3.3/css/#unstyled):

image

File: modules/core/server/views/partials/footer.server.view.html [hunk 4]
```diff
@@ -2,31 +2,47 @@
...
+        <li><a ui-sref="privacy">Privacy</a></li>
+        <li><a ui-sref="contact">Contact</a></li>
+        <li><a ui-sref="faq">FAQ</a></li>
+        <li><a ui-sref="volunteering">Volunteering</a></li>
+        <li><a ui-sref="rules">Rules</a></li>
+        <li><a ui-sref="rules">Rules</a></li>
+        <li><a ui-sref="donate">Donate</a></li>
+      </ul>
+    </div>
+    <div class="col-xs-6 pull-right">
```
Comment (simison) [inline modules/core/server/views/partials/footer.server.view.html]: I don't think `pull-right` has any effect here?

File: modules/core/server/views/partials/footer.server.view.html [hunk 5]
```diff
@@ -5,25 +5,39 @@
...
-        <li><a ui-sref="rules">Rules</a></li>
-        <li><a ui-sref="donate">Donate</a></li>
       </ul>
     </div>
-    <div class="col-xs-6 pull-right">
+    <div class="col-xs-6">
       <ul class="list-inline">
+        <li>
+          {% if appSettings.commit %}
+          {% if appSettings.commitTime %}Last change {{ appSettings.commitTime }}{% endif %}
```
Comment (simison) [inline modules/core/server/views/partials/footer.server.view.html]: Honestly to me this is still just noise and I think there are better ways to attract volunteers, but if you really want it there... :-D

Comment (Akronix) [reply modules/core/server/views/partials/footer.server.view.html]: Discussion:
https://meta.trustroots.org/t/show-last-date-the-code-was-changed/87

**Comment by simison** [comment]
CI-failures seem unrelated; could you try if rebasing helps?

**Pull request merged**